Utilities/Hooks/kw-commit-msg

```diff
@@ -1,123 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
This deletion should probably have been part of the previous commit.

Reply:
the file is needed the other PR if this one is rejected.